app/helpers/kaui/date_helper.rb

```diff
@@ -13,7 +13,7 @@ def format_date(date, timezone="Pacific Time (US & Canada)")
 
       # If not, convert into account timezone and return the date part only
       parsed_date = DateTime.parse(date.to_s).in_time_zone(timezone)
-      parsed_date.to_s(:date_only)
+      parsed_date.to_s(:pretty)
```

Comment:
Hi Pierre, if KAUI is used for some support guys, they want to get the exact time. Is it better to create repo in company and customize it ?